patch-image.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 PATCH_FILE="/tmp/${PATCH_LIST}"
 VARS="PROJECT REFSPEC GIT_HOST"
 
-dnf install -y python3-pip
+dnf install -y python3-pip git-core
```

Comment:
Should we remove the packages at the end of script, just like `BUILD_DEPS` are?

Reply:
I will remove git core, but I don't want to remove python3-pip in this PR as it was like this before too.

Reply:
@elfosardo your opinion on it?

Reply:
I have removed both(python3-pip git-core) now. I have tested and it didn't cause any issue so I think we are good to go. I have updated the commit kindly check.